__main__.py

```diff
@@ -4,43 +4,99 @@
 Usage: run the exerciser with a variety of options
 """
 
-from pathlib import Path
 import argparse
-from datetime import datetime
 import sys
 import os
 
 sys.path.append(os.path.join(os.path.dirname(__file__), "src"))
 from src import general
 
+
 def parse_cla() -> argparse.Namespace:
     """
     Usage: command line argument parser
     @return: parsed arguments in argparse.Namespace object
     """
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("test_name",
-                        type=str,
-                        metavar="test_name",
-                        choices=["sleep_test", "checksum_test"],
-                        help="name of test to be run by exerciser")
+    parser.add_argument(
+        "tests",
+        nargs="*",
+        default=[],
+        help="specify the test/s to run in a space seperated list. if no tests "
+        + "are listed, the exerciser will run all available tests.",
```

Comment:
Consider the following description: `Comma separated list of specific tests to execute. If not specified all available tests are executed.`